modules/objdetect/include/opencv2/objdetect.hpp

```diff
@@ -801,71 +840,91 @@ class CV_EXPORTS_W QRCodeDetector
-
-     Returns UTF8-encoded output string or empty string if the code cannot be decoded.
-     @param img grayscale or color (BGR) image containing QR code.
-     @param points Quadrangle vertices found by detect() method (or some other algorithm).
-     @param straight_qrcode The optional output image containing rectified and binarized QR code
-     */
-    CV_WRAP cv::String decodeCurved(InputArray img, InputArray points, OutputArray straight_qrcode = noArray());
-
     /** @brief Both detects and decodes QR code
 
      @param img grayscale or color (BGR) image containing QR code.
      @param points optional output array of vertices of the found QR code quadrangle. Will be empty if not found.
      @param straight_qrcode The optional output image containing rectified and binarized QR code
      */
-    CV_WRAP std::string detectAndDecode(InputArray img, OutputArray points=noArray(),
+    CV_WRAP std::string detectAndDecode(InputArray img, OutputArray points = noArray(),
                                         OutputArray straight_qrcode = noArray());
 
-    /** @brief Both detects and decodes QR code on a curved surface
-
-     @param img grayscale or color (BGR) image containing QR code.
-     @param points optional output array of vertices of the found QR code quadrangle. Will be empty if not found.
-     @param straight_qrcode The optional output image containing rectified and binarized QR code
-     */
-    CV_WRAP std::string detectAndDecodeCurved(InputArray img, OutputArray points=noArray(),
-                                              OutputArray straight_qrcode = noArray());
 
     /** @brief Detects QR codes in image and returns the vector of the quadrangles containing the codes.
      @param img grayscale or color (BGR) image containing (or not) QR codes.
      @param points Output vector of vector of vertices of the minimum-area quadrangle containing the codes.
      */
-    CV_WRAP
-    bool detectMulti(InputArray img, OutputArray points) const;
+    CV_WRAP bool detectMulti(InputArray img, OutputArray points) const;
 
     /** @brief Decodes QR codes in image once it's found by the detect() method.
      @param img grayscale or color (BGR) image containing QR codes.
      @param decoded_info UTF8-encoded output vector of string or empty vector of string if the codes cannot be decoded.
      @param points vector of Quadrangle vertices found by detect() method (or some other algorithm).
      @param straight_qrcode The optional output vector of images containing rectified and binarized QR codes
      */
-    CV_WRAP
-    bool decodeMulti(
-            InputArray img, InputArray points,
-            CV_OUT std::vector<std::string>& decoded_info,
-            OutputArrayOfArrays straight_qrcode = noArray()
-    ) const;
+    CV_WRAP bool decodeMulti(InputArray img, InputArray points, CV_OUT std::vector<std::string>& decoded_info,
+                             OutputArrayOfArrays straight_qrcode = noArray()) const;
```

Comment:
Please keep the existed coding style.
No need to introduce unnecessary changes in this patch.

To keep code readable and maintainable, prefer to:
- put `CV_WRAP` and other modifiers (static/+inline) on a separate line (that could be modified frequently to support different bindings)
- use dedicated line for method/function name in cases with long list of parameters.

Reply:
fixed